### ISSUE TITLE
Does hyperscript supports variables?

### ISSUE THREAD

**Issue by Nicero** [bigskysoftware/_hyperscript#105]
Could someone kindly answer a [question](https://stackoverflow.com/questions/66892328/how-to-use-variables-in-hyperscript-js) I posted on Stackoverflow asking if I can use variables with the `add` command?

For example:
without variable (id = #1022):
```
<button _="on click add .green to #1022" type="button">
    Click
</button>
```
with variable (id = my_variable):
```
<button _="on click add .green to my_variable" type="button">
    Click
</button>
```
and then something like this:
```
<script type="text/javascript">
    function setID() {
        // here the code to get the ID and set my_variable
        return id;
    }
</script>
```

**Comment by 1cg** [comment]
Hi @Nicero, thank you for trying out hyperscript.  I answered your question on Stack Overflow (thank you for asking it there!)

I realized afterwards that the answer that I gave will only work in the dev version of hyperscript!  Sorry about that, the language is still very early and is evolving quickly.  Are you willing to work against the dev version of the language?  I hope to cut a 0.0.9 release next week, which will include the feature needed (templates in query literals).  I also realized that it could be simplified to this:

```html
<button _="on click add .green to <$my_variable/>" type="button">
    Click
</button>
```

If you are willing to jump on the htmx/hyperscript discord, we can coordinate with you:

htmx.org/discord

thank you again for taking a look at hyperscript!
